service/src/main/java/uk/gov/hmcts/reform/fpl/service/HearingBookingService.java

```diff
@@ -50,4 +51,12 @@ public HearingBooking getHearingBooking(final List<Element<HearingBooking>> hear
             .map(Element::getValue)
             .orElse(HearingBooking.builder().build());
     }
+
+    public HearingBooking getHearingBookingByUUID(List<Element<HearingBooking>> hearingBookings, UUID elementId) {
```

Comment:
Not sure if the service should return the "default" empty one, it's might be confusing, would prefer this to return the optional and caller building the object.

Reply:
This will change once merged with FPLA-929. Myself and Sam shared the same method.

Reply:
It has been replaced to return null in #675

Reply:
yeah, I don't even have tests for this 👀 . Lifted from James' PR, obviously from an older iteration